pandas/core/frame.py

```diff
@@ -20,6 +20,7 @@
     Iterator,
     Mapping,
     Sequence,
+    Set as AbstractSet,
```

Comment:
Why the alias?

Reply:
Also, can you just use `set` instead of importing from `abc`?

Reply:
The alias was done by a hook when I tried to import Set from `abc` since it was ambiguous with `set`. I like your suggestion of just using `set` instead. Updated.